### ISSUE TITLE
Track change failed

### ISSUE THREAD

**Issue by Artioss** [yanpanlau/DDPG-Keras-Torcs#6]
I trained a model in the track CG Speedway number 1, and it work well when test in this track. But when I change the track(e.g. to CG track 2),  and the trained model seem not work, the car run out of the track. Is that means for every different track, I have to train different models ?

**Comment by yanpanlau** [comment]
No. It probably means you are overfit the model in CG Speedway. I suggest you try following

1) Train on a more difficult track and test out in CG speedway number 1 instead.
2) Apply L2 Regularization to your Neural Network and make sure not overfit
3) Do cross-validation on training/testing different tracks.

In fact, even in the original Deepmind paper, they did not test their model on the validation data. I did validate my model where the training data and validation dataset are different.

**Comment by Artioss** [comment]
@yanpanlau  Thanks for your suggestions, i'll try it soon. Could you please tell me what's your training & validation tracks?
By the way, what's your opinion with issue #2?
 Thank you.

**Comment by deepmeng** [comment]
would you like tell us how you change the track by which interface or script?
@yanpanlau @Artioss

**Comment by yanpanlau** [comment]
Hi,

1) sudo torcs
2) Click Race
3) Click Practice
4) Click Configure Race
5) Choose different track
6) Click New Race

Then you should see the game load a new track. Then you can click Ctrl + C
to quit and run your AI agent.

**Comment by deepmeng** [comment]
Hi man, sorry to bother you, this visualization method we all know. Do you know is there a method to config it in some functions? I read and modify the *.py files, but it does not work.

**Comment by yanpanlau** [comment]
I have not do it myself before. The original author (gym_torcs) use a unix
tools called xautomation to do the "actual mouse click", you may want to
try that

**Comment by deepmeng** [comment]
thanks very much!

**Comment by deepmeng** [comment]
Hi man, excuse me, could you please tell us how we can change the mode from "practice" to "quick race", and always keep it in "quick race"?  THX very much!

**Comment by JiaojiaoYe1994** [comment]
Hey, I currently also look into this project, have you found how to change the mode and could you please tell me?
 @deepmeng

**Comment by deepmeng** [comment]
@JiaojiaoYe1994 I'm very sorry about this. I can't remember it anymore...
